assets/sass/vendor/_mdc-dialog.scss

```
@@ -48,23 +42,20 @@
 
 		svg {
 			color: $c-red-r-500;
+			margin-bottom: 4px;
 
-			@media (max-width: $bp-mobileOnly) {
-				margin-bottom: 4px;
+			@media (min-width: $bp-tablet) {
+				margin-bottom: 0;
 			}
 		}
 	}
 
 	.mdc-dialog__lead {
-		font-size: $fs-title-sm;
-		font-weight: $fw-medium;
+		font-size: $fs-label-md;
+		font-weight: $fw-normal;
```

This should be `$fw-medium` as the font-weight is 500.

```suggestion
		font-weight: $fw-medium;
```